Src/Sarif.PatternMatcher/SearchSkimmer.cs

```diff
@@ -292,23 +292,27 @@ private void RunMatchExpression(FlexMatch binary64DecodedMatch, AnalyzeContext c
                             // The validator returned a bad value.
                             // TODO: we should log this condition
                             // and then continue processing.
+                            level = FailureLevel.Error;
                             break;
                         }
 
                         case Validation.Valid:
                         {
+                            level = FailureLevel.Error;
                             validationState = " which was determined to be valid";
                             break;
                         }
 
                         case Validation.Invalid:
                         {
+                            level = FailureLevel.Warning;
                             validationState = " which was determined to be invalid";
                             break;
                         }
 
                         case Validation.InvalidForConsultedAuthorities:
```

Comment:
>InvalidForConsultedAuthorities [](start = 40, length = 30)

We did some checking and it *looks* invalid, so, we'll set it to a warning. This assumes the user has configured the scan to check for the most critical authorities.